benches/allocs.rs

```diff
@@ -0,0 +1,27 @@
+use rand::Rng;
+
+use vart::art::Tree;
+use vart::VariableSizeKey;
+
+#[global_allocator]
+static ALLOC: divan::AllocProfiler = divan::AllocProfiler::system();
+
+fn main() {
+    divan::main();
+}
+
+const COUNTS: &[usize] = &[10_000, 100_000, 1_000_000];
```

Comment:
If we are only interested in allocations, then having only one count, let's say 1_000_000, should be enough.

Reply:
Fixed